contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Organizations and projects
+sidebar: Docs
+showTitle: true
+---
+
+For a user experience that scales for all your needs, PostHog implements the concepts of organizations and projects.
```

Comment:
> or a user experience that scales for all your needs

This doesn't tell me why I want these concepts. Can we provide a better overview of why Organizations and Projects are valuable?

Reply:
Very vague, agreed. I kept it very concise, but explained better now